### PR TITLE
Notification for password change

### LINKED ISSUE #723: No notification is shown after successful password change
@deshraj @aka-jain if this sounds good enough. I would like to work on this.

### PR DESCRIPTION
**Pull request by anweshknayak**
@deshraj  @aka-jain please check

fixes #723

### CODE REVIEW

File: frontend/src/js/controllers/changePwdCtrl.js [hunk 1]
```diff
@@ -55,6 +55,7 @@
                         vm.user.error = false;
                         console.log("PASSWORD CHANGED SUCCESSFULLY");
```
Comment (spyshiv) [inline frontend/src/js/controllers/changePwdCtrl.js]: @anweshknayak : Please remove these console.log();

**Comment by anweshknayak** [comment]
@spyshiv done. check #722 too related to this only.

**Comment by spyshiv** [comment]
Ok. Let me check

File: frontend/src/js/controllers/changePwdCtrl.js [hunk 2]
```diff
@@ -53,8 +53,8 @@
                         var status = response.status;
                         var response = response.data;
                         vm.user.error = false;
-                        console.log("PASSWORD CHANGED SUCCESSFULLY");
                         console.log(response);
```
Comment (spyshiv) [inline frontend/src/js/controllers/changePwdCtrl.js]: Great work :+1:
@anweshknayak: remove this console too. Then i'll merge this PR.
LGTM.

**Comment by anweshknayak** [comment]
@spyshiv done. please review

File: frontend/src/js/controllers/changePwdCtrl.js (hunk 2 above)
Comment (anweshknayak) [inline frontend/src/js/controllers/changePwdCtrl.js]: Thanks :)

**Comment by spyshiv** [comment]
cool. :+1: 
merging this.

**Pull request merged**